addOns/postman/src/test/java/org/zaproxy/addon/postman/PostmanParserUnitTest.java

```diff
@@ -71,4 +181,70 @@ void shouldIgnoreUnKnownAttributes() throws Exception {
         String collectionJson = "{\"unKnown1\":true,\"unKnown2\":\"\"}";
         assertDoesNotThrow(() -> parser.parse(collectionJson));
     }
+
+    @ParameterizedTest
+    @MethodSource("extractionTestData")
+    void shouldExtractHttpMessagesFromItems(List<AbstractItem> items, int numberOfitems)
+            throws Exception {
+        List<HttpMessage> httpMessages = new ArrayList<>();
+        PostmanParser.extractHttpMessages(items, httpMessages);
+
+        assertEquals(numberOfitems, httpMessages.size());
+    }
+
+    // The 'Content-Type' header gets set according to the mode of the request body, but if it's
+    // explicitly defined in the request, that value will take precedence
+    void shouldNotSetContentTypeIfExplicitlySet() {
```

Comment:
Not actually a test, on purpose?

Reply:
Didn't get. It is testing if the given content type is not getting overridden by the actual one.

Reply:
Does not have the `@Test` annotation.